config/generateRunFile.coffee

```diff
@@ -442,6 +442,72 @@ generateDev = (KONFIG, options) ->
       fi
     }
 
+    function k8s () {
+      params=(start stop)
+      param=$1
+      case "${params[@]}" in  *"$param"*)
+        ;;
+      *)
+        echo "Error: Command not found: $param"
+        echo "Usage: ./run k8s COMMAND"
+        echo ""
+        echo "Commands:  "
+        echo "  start : install the services in Kubernetes pods"
+        echo "  stop  : stop the currently running Kubernetes cluster"
+
+        echo ""
+        exit 1
+      ;;
+      esac
+
+      if [ "$param" == "start" ]; then
+        command -v VirtualBox       >/dev/null 2>&1 || { echo >&2 "I require curl but it's not installed.  Aborting."; exit 1; }
+        command -v minikube         >/dev/null 2>&1 || { echo >&2 "I require a Kubernetes cluster. To install minikube: \
+                (curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.20.0/minikube-darwin-amd64 && chmod +x minikube && mv minikube /usr/local/bin/)"; exit 1;}
+        command -v kubectl          >/dev/null 2>&1 || { echo >&2 "I require kubectl. To install kubectl: \
+                (curl -L -O https://storage.googleapis.com/kubernetes-release/release/v1.6.4/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/)"; exit 1;}
+        minikube start
+
+        if [[ `uname` == 'Darwin' ]]; then
+          command -v helm           >/dev/null 2>&1 || { echo >&2 "I require kubernetes-helm but it's not installed. (brew install kubernetes-helm maybe?)"; exit 1; }
```

Comment:
you can advise this https://github.com/kubernetes/helm/blob/master/docs/install.md#from-script